js/b2w-redirection-ajax.js

```diff
@@ -40,7 +40,14 @@ function rt_start_config( nonce ){
 function generate_code( domain_name, curr_domain, nonce){
 
 	jQuery( '#code_here' ).show();
-	var response = jQuery( '#code_here' ).html();
+	let originalHtml = jQuery('#code_here').data('original-html');
+
+	if (!originalHtml) {
+		originalHtml = jQuery('#code_here').html();
```

Comment:
Please avoid using the `html()` function.
Ref: https://docs.wpvip.com/security/javascript-security-recommendations/

Reply:
Hi @Pathan-Amaankhan , Thanks for the review.

I'm working on making this plugin vip compatible and solving the above issue in #47 PR.
This PR only focus on solving the #42 issue and that's why I didn't change the `html( )` function in existing code here.